### PR TITLE
(maint) Fix bundle install directory

### PR DESCRIPTION
**Pull request by GabrielNagy**

### CODE REVIEW

**Comment by mihaibuzgau** [comment]
jenkins please test this on fedora30-64a,osx1015-64a,windows2019-64a,ubuntu1604-32a

**Comment by mihaibuzgau** [comment]
jenkins please test this on fedora30-64a,osx1015-64a,windows2019-64a,ubuntu1604-32a

File: CMakeLists.txt [hunk 1]
```diff
@@ -186,7 +186,8 @@ add_test(NAME "libfacter\\ tests" COMMAND libfacter_test)
 if (RUBY_FOUND)
     find_program(BUNDLER_PATH NAMES bundle.bat bundle)
     if (BUNDLER_PATH)
-        execute_process(COMMAND ${BUNDLER_PATH} --retry 2 --gemfile=lib/Gemfile --quiet)
+        message(STATUS "Bundler found, installing dependencies for Ruby tests...")
+        execute_process(COMMAND ${BUNDLER_PATH} --retry 2 WORKING_DIRECTORY "${PROJECT_SOURCE_DIR}/lib")
```
Comment (melissa) [inline CMakeLists.txt]: Does this need to be `COMMAND ${BUNDLER_PATH} install`?

Comment (GabrielNagy) [reply CMakeLists.txt]: AFAIK, running bundler with no argument should default to bundle install, but it doesn't hurt to be more verbose...

Comment (mihaibuzgau) [reply CMakeLists.txt]: updated on my PR: https://github.com/puppetlabs/facter/pull/1880

**Comment by mihaibuzgau** [comment]
Closing in favor of: https://github.com/puppetlabs/facter/pull/1880

**Pull request closed without merging**